development/components/form/types-reference/unit-type-extension.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: TextWithUnitType
+title: UnitTypeExtension
```

Comment:
Since the original article was changed, consider adding an alias
```suggestion
title: UnitTypeExtension
aliases:
  - /8/development/components/form/types-reference/text-with-unit/
```